components/Orders.tsx

```diff
@@ -0,0 +1,25 @@
+import { List } from "antd";
+import { useSelector, useDispatch } from "react-redux";
+
+export default function Basket() {
+  const dispatch = useDispatch();
+
+  const orders = useSelector((state: any) => state.orders.items) as {
```

Comment:
You can use `types/global.d.ts` to define global types, eg. `type OrderItem = { url: string }`. It would be more readable here to use `OrderItem[]`.